components/attendance/AttendanceSelector.tsx

```diff
@@ -0,0 +1,92 @@
+import cn from "@/utils/helpers/cn";
+import {
+  AbsenceType,
+  AttendanceEvent,
+  StudentAttendance,
+} from "@/utils/types/attendance";
+import { StylableFC } from "@/utils/types/common";
+import { ChipSet, InputChip } from "@suankularb-components/react";
+import { useTranslation } from "next-i18next";
+
+const AttendanceSelector: StylableFC<{
```

Comment:
It’s good practice to always document components with [JSDoc](https://jsdoc.app/about-getting-started), despite how obvious the component may seem to you, because it might not be obvious to others!

GitHub Copilot or any other LLM helps a lot with JSDoc, but always double-check that its output matches your intention and convention made by other components.